Documentation/bash-scripts.md

```diff
@@ -0,0 +1,52 @@
+# Bash scripts
+
+A common way to use acbuild is via a bash script. There's a common template
+that the examples for acbuild follow, and it makes cleanup a little easier. The
+template is at the end of this file, and here we explain the reasoning behind
+the parts of it.
+
+The 1st line, `#!/usr/bin/env bash`, is simply good form to make sure the
```

Comment:
Since the script leverages `bash` features, we open it by specifying execution in the `bash` shell, rather than just inheriting the user's `SHELL` enviroment.

Further, we set the `-e` option to `bash` to ensure that the entire script exits on the failure of any command. This gives us some atomicity, ensuring that either a complete and valid ACI is constructed, or none at all.